docs/testing-components/screenshot-testing/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Screenshot Testing
+---
+
+## What is the Screenshot Testing
+Screenshot test is visual test that takes a screenshot, then compares it to a reference screenshot file saved. When ui changes, one of the major considering is whether visual changes exist. Some changes may be easy to check, but some changes are hard to check due to very small change. With Screenshot test, very small changes can easily check due to compare pixel-by-pixel. Additionlally, Screenshot testing is the most reliable test for static images. If you want to test animated image or video, you should check result.
```

Comment:
```suggestion
Screenshot test is visual test that takes a screenshot, then compares it to a reference screenshot file saved. When ui changes, one of the major considerations is whether visual changes exist. Some changes may be easy to check, but some changes are hard to check due to very small change. With Screenshot test, very small changes can easily check due to compare pixel-by-pixel. Additionally, Screenshot testing is the most reliable test for static images. If you want to test animated image or video, you should check the result.
```

I don't understand your intend for the last sentence

I don't know your intend for last sentence
"If you want to test animated image or video, you should check result."